src/index.html

```diff
@@ -73,22 +86,36 @@
         <div class="navi-right float-right">
           <ul>
             <li>
-              <a href="#"
-                ><i class="fa fa-user-circle"></i><span>Sign in</span></a
-              >
+               <div id="root"></div>
```

Comment:
Please use a unique ID to render your component. 
```suggestion
               <div id="signId"></div>
```